rasa/utils/tensorflow/tf_models.py

```diff
@@ -17,7 +17,7 @@ class RasaModel(tf.keras.models.Model):
     Cannot be used as tf.keras.Model
     """
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, random_seed: Optional[int], *args, **kwargs):
```

Comment:
I don't think we need `*args` here. Let's set `random_seed` to `None` by default